DTS-STN/Service-Canada-Design-System
# fix: changed alt text on warning icon

# DS-##(82676)

Main Reviewer: @zeddotes 

#### Description & Background Context:

Dan Villeneuve noted we should change the alt text for the Warning icon used in the `Message.js` component and CallOut storyExample from "Warning icon" to "Warning".

#### Where should the reviewer start?

`src/components/Message.js`
`src/storyExamples/CallOut/CallOut.js`

#### How should this be tested?

1. npm run storybook
2. See that `alert_icon_alt_text` property value in Message component = "Warning"

#### Are there any breaking changes?

No
